merlin/models/tf/blocks/core/transformations.py

```diff
@@ -436,6 +436,51 @@ def call(self, inputs, training=False, **kwargs) -> tf.Tensor:
         return logits
 
 
+@Block.registry.register_with_multiple_names("categorical_to_onehot")
```

Comment:
This won’t work to expose it as a string since this class in it’s current form requires an input argument (`cardinalities`). It would be better to refactor it a little bit and infer the cardinalities from the Schema. There’s a way to have this class be passed the schema automatically. See [this](https://github.com/NVIDIA-Merlin/models/blob/main/merlin/models/tf/blocks/core/aggregation.py#L237) transformation on how to do that.

Reply:
Right, thanks for the example! I updated the class accordingly.